content/2025/_index.md

```diff
@@ -19,13 +19,13 @@ layout: home
                 url="technical/call"
                 icon="right" >}}
```

Comment:
I suggest to change 
* this label to "Call for Papers"
* this icon to "cfp" as this stands for "call for paper"